clients/issue.go

```diff
@@ -18,6 +18,15 @@ import "time"
 
 // Issue represents a thread like GitHub issue comment thread.
 type Issue struct {
-	URI       *string
-	UpdatedAt *time.Time
+	URI               *string
+	CreatedAt         time.Time
+	ClosedAt          time.Time
+	AuthorAssociation string
```

Comment:
Update `time.Time` -> `*time.Time` and `string` -> `*string`. We are in the process of moving to a convention where we can use `nil` values to represent no data instead of relying on default values. See #1242 for context.